src/transformers/models/big_bird/modeling_big_bird.py

```diff
@@ -2082,7 +2082,7 @@ def forward(
             masked_lm_loss = loss_fct(prediction_scores.view(-1, self.config.vocab_size), labels.view(-1))
 
         if not return_dict:
-            output = (prediction_scores,) + outputs[1:]
+            output = (prediction_scores,) + outputs[2:]
```

Comment:
@vasudevgupta7 - this was the reason the `test_model_outputs_equivalence` didn't pass -> Bert-like models return the `pooled_output` at the 2nd position -> so in order to get the attentions we need to pass `ouputs[2:]` here